Misc/NEWS.d/next/Library/2021-05-25-01-43-53.bpo-44222.mwqYlK.rst

```diff
@@ -0,0 +1,2 @@
+Improve the performance of _removeHandlerRef() for very long _handlerLists.
```

Comment:
I've added the `skip-news` label, so this file shouldn't be necessary (it's too low-level a detail for the NEWS file). You can remove this, your name should appear as author in the commit message.

Reply:
Still haven't got the hang of when we add NEWS entry and when we don't :disappointed:  (in previous PRs I've added NEWS files for even lower-level changes, such as 350526105fa9b131d8b941ae753378b741dabb2f. Perhaps because they were bugfixes and not just improvements?)

Anyway - sure, removing it